README.md

```diff
@@ -149,6 +149,21 @@ below command.
 
 ```
 sudo ovnkube -k8s-kubeconfig kubeconfig.yaml -loglevel=4 \
+    -logfile="/var/log/openvswitch/ovnkube.log" \
+    -k8s-apiserver="http://$CENTRAL_IP:8080" \
+    -init-node="$NODE_NAME"  \
+    -nodeport \
+    -nb-address="tcp://$CENTRAL_IP:6631" \
+    -sb-address="tcp://$CENTRAL_IP:6632" -k8s-token="$TOKEN" \
+    -init-gateways \
+    -service-cluster-ip-range=$SERVICE_IP_SUBNET \
+    -cluster-subnet=$CLUSTER_IP_SUBNET 2>&1
+```
+
+You can run the above command in the background instead, with:
```

Comment:
Just keep the one with 'nohup'. Its a repeat, right?

Reply:
I wonder whether people will miss out any obvious error if they send it to background without passing the correct parameters.

Reply:
I think a comment to check the logfile may be useful either which way.